src/lib/modules/ModuleContainer/index.js

```diff
@@ -0,0 +1,37 @@
+class ModuleContainer {
+  constructor(io, { modules = [] }) {
+    this.io = io;
+    this.handle = this.handle.bind(this);
+    this.registeredModule = modules;
+    this.activatedModule = [];
+  }
+
+  activate(name) {
+    const module = this.registeredModule.find(item => item.name === name);
```

Comment:
A huge part of this function is duplicated in `deactivate`, maybe we would factor a bit ? 😊